### PR TITLE
Add video blog post about KRaft

### PR DESCRIPTION
**Pull request by scholzj**
As with the previous PRs, this PR adds a new _video_ blog post about the KRaft video which is planned to be published tomorrow.

### CODE REVIEW

File: _posts/2022-05-26-zookeeper-less-kafka-in-strimzi-0.29.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title:  "Video: ZooKeeper-less Kafka in Strimzi 0.29?"
```
Comment (ppatierno) [inline _posts/2022-05-26-zookeeper-less-kafka-in-strimzi-0.29.md:3]: I would use the complete versioning as we do in other blog posts, so 0.29.0.

File: _posts/2022-05-26-zookeeper-less-kafka-in-strimzi-0.29.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+date: 2022-05-25
+author: jakub_scholz
+---
+
+One of the new features introduced in Strimzi 0.29 is a new feature gate `UseKRaft`.
```
Comment (ppatierno) [inline _posts/2022-05-26-zookeeper-less-kafka-in-strimzi-0.29.md:8]: Ditto as above.

File: _posts/2022-05-26-zookeeper-less-kafka-in-strimzi-0.29.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+This feature gate provides a way to deploy a Kafka cluster in the KRaft (Kafka Raft metadata) mode without ZooKeeper.
+This feature gate is currently experimental only and is intended to be used mainly used for development and testing of Strimzi and Apache Kafka.
```
Comment (ppatierno) [inline _posts/2022-05-26-zookeeper-less-kafka-in-strimzi-0.29.md:10]: "to be mainly used" ... "used" is used ;-) ... too many times

**Review by ppatierno — APPROVED** [review]
LGTM. I left some comments but no need to re-review after fixing them.

**Review by PaulRMellor — APPROVED** [review]
LGTM :+1:

**Pull request merged**